source/include/vm.hpp

```diff
@@ -19,6 +19,20 @@
 
 #define VM_FULL_DEBUG
 
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//
+// V2
+//
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+namespace ExVM {
```

Comment:
This is the main purpose of the PR: Create a new namespace and rearrange the existing bits within it.